src/Dropdown.elm

```diff
@@ -4,6 +4,7 @@ module Dropdown exposing
     , Config, basic, filterable
     , withContainerAttributes, withPromptElement, withFilterPlaceholder, withSelectAttributes, withSearchAttributes, withOpenCloseButtons, withListAttributes
     , update, view
+    , CustomBasicConfig, custom
```

Comment:
Hey, do you mind adding this in the documentation comment tag as well? as we are exposing this type, it needs to have comments, otherwise we will not able to release a new version.